src/assets/scss/components/_oktaSigninWidget.scss

```
@@ -194,6 +194,11 @@ sign-up-modal {
         border-bottom-left-radius: 0;
       }
 
+      /* stylelint-disable-next-line selector-class-pattern */
+      .o-form-input-name-userProfile\.zipCode ~ .o-form-input-error {
```

What about this for consistency with some of the other selectors?

It may even make sense to move the border style to this existing rule:

```scss
&[data-se="o-form-fieldset-userProfile.zipCode"] {
  border-left: 1px solid $okta-grey;
}
```

```suggestion
      &[data-se="o-form-fieldset-userProfile.zipCode"] .o-form-input-error {
```